packages/patternfly-4/react-core/src/components/EmptyState/EmptyStateBody.tsx

```diff
@@ -14,7 +14,7 @@ export const EmptyStateBody: React.FunctionComponent<EmptyStateBodyProps> = ({
   className = '',
   ...props
 }: EmptyStateBodyProps) => (
-  <p className={css(styles.emptyStateBody, className)} {...props}>
+  <div className={css(styles.emptyStateBody, className)} {...props}>
```

Comment:
I think HTMLParagraphElement on line 5 needs to be changed to HTMLDivElement.

Reply:
nice catch!